### DISCUSSION TITLE
Aroon Up/Down and Oscillator

### DISCUSSION THREAD

**Discussion by DaveSkender** [DaveSkender/Stock.Indicators#266] [Indicators]
Community discussion about [Aroon Up/Down and Oscillator](https://dotnet.stockindicators.dev/indicators/Aroon/#content)

**Comment by GmiSoft** [comment]
Hi Dave,

I have a question about the interpretation of the Oscillator property of the Aroon indicator.
Thank you in advance.

**Reply by DaveSkender** [reply]
Sure, what's your question?

**Reply by GmiSoft** [reply]
I found in literature information about how to use the Aroon indicator for buying or selling signals (using Up and Down values). But no information on how to use the Oscillator value. maybe I missed something ^^

**Reply by DaveSkender** [reply]
The oscillator is an optional or alternate component and is simply `AroonUp-AroonDown`, so its another way of looking at the combined trend of both lines.  StockChart.com has a good explanation of the oscillator [here](https://school.stockcharts.com/doku.php?id=technical_indicators:aroon_oscillator).
